liblangutil/Exceptions.h

```diff
@@ -189,6 +191,19 @@ class Error: virtual public util::Exception
 		return "Error";
 	}
 
+    static std::optional<Severity> stringToSeverity(std::string _severity)
+	{
+        _severity = tolower(_severity);
```

Comment:
`tolower()` works on `char`, not `string`. You can use `boost::algorithm::to_lower()` instead.

But please move the body of this function to `liblangutil/Exceptions.cpp`. This header is included by a lot of other files so it's better not to have boost stuff in it.

Reply:
Fixed 👍